addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -541,6 +541,17 @@ export class Wysiwyg extends Component {
         );
 
         this.$editable.on('click', '.o_image, .media_iframe_video', e => e.preventDefault());
+        this.$editable.on('click', '.o_editor_banner > i, .o_editor_banner > span', event => {
+            this.popover.add(event.target, EmojiPicker, {
+                /**
+                 * @param {string} emoji
+                 */
+                onSelect: emoji => {
+                    const $icon = $(`<span class="mb-3">${emoji}</span>`);
```

Comment:
Note that it does not respect the existing structure of a "i" tag and makes it so we loose the aria-label.

Reply:
Note that ideally we could modify the existing code to use actual emojis instead of fa-classes, maybe it would make the transition easier, what do you think?

Reply:
Ok, I will also use a `i` tag for the emojis instead of a `span` :smiley:

For the aria-label, I don't see any `aria-label` on the default icon of the banner. I don't know what can I provide as `aria-label` for the emojis as the emoji picker does not provide an actual description of the selected emoji.

For the default emoji of the banners, I'm not sure which emoji to use for every banner. If we change the default icons, we will probably have to update the existing banners, write migration scripts, etc. I would say that If Luc is happy with the current implementation, we can keep it as is.

Reply:
I don't really like that what is displayed for you originally is not something selectable inside the emojis box, it feels strange.
Meaning if you select an emoji, you can't go back to what you had before.

Surely we can find 4 appropriate emojis to replace the current fa icons, right?